job-dsl-plugin/build.gradle

```diff
@@ -22,9 +22,7 @@ jenkinsPlugin {
     displayName = 'Job DSL'
     url = 'https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin'
     gitHubUrl = 'https://github.com/jenkinsci/job-dsl-plugin'
-    maskClasses = 'groovy.grape'
     compatibleSinceVersion = '1.35'
-    pluginFirstClassLoader = true
```

Comment:
I already did that. 1.565 uses Guava 11.0.1 and Groovy 1.8.9. See https://github.com/jenkinsci/jenkins/blob/jenkins-1.565/pom.xml#L181 and https://github.com/jenkinsci/jenkins/blob/jenkins-1.565/core/pom.xml#L47.

I'm planning to remove the dependency to Guava to avoid problems. Must of the utility stuff is also in the Groovy library and in job-dsl-core are only about 10 usages left.